pkgs/development/compilers/llvm/11/default.nix

```diff
@@ -72,12 +72,26 @@ let
       inherit clang-tools-extra_src llvm_meta;
     };
 
+    libllvm-polly = callPackage ./llvm {
+      inherit llvm_meta;
+      enablePolly = true;
+    };
+
+    llvm-polly = tools.libllvm-polly.lib // { outputUnspecified = true; };
```

Comment:
so the cross-version diff is minimized, can this be moved up? I was comparing the overall default.nix for LLVM 7, fwiw.